.pre-commit-config.yaml

```diff
@@ -6,26 +6,27 @@ default_language_version:
 
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.5.0
+    rev: v4.4.0
     hooks:
     - id: check-yaml
     - id: check-ast
     - id: trailing-whitespace
     - id: check-toml
   - repo: https://github.com/PyCQA/isort
-    rev: 5.13.1
+    rev: 5.12.0
     hooks:
       - id: isort
+        args: ["--profile=black"]
   - repo: https://github.com/PyCQA/flake8
     rev: 6.1.0
     hooks:
       - id: flake8
   - repo: https://github.com/psf/black
-    rev: 23.11.0 # Replace by any tag/version: https://github.com/psf/black/tags
+    rev: 23.9.1 # Replace by any tag/version: https://github.com/psf/black/tags
     hooks:
       - id: black
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.15.0
+    rev: v3.10.1
```

Comment:
Same question, just looking to understand, or I may be misreading

Reply:
Bumped it back up for merge conflict.

Reply:
They still look lower to me, why are they lesser versions

Reply:
I had some weirdness with GitHub, bumped it back up.